src/read.jl

```diff
@@ -104,113 +104,160 @@ function read_storage(filepath)::Storage
 end
 
 
-"""Load demand flexibility profile from .csv files into DataFrame(s)."""
+"""
+    read_demand_flexibility(filepath, interval)
+
+Load demand flexibility profiles and parameters from .csv files and return them in a 
+DemandFlexibility struct.
+"""
 function read_demand_flexibility(filepath, interval)::DemandFlexibility
     # Initialize demand flexibility
     demand_flexibility = Dict()
 
-    # Try loading demand flexibility profiles. Otherwise, create a DataFrame of zeros
+    # Pre-specify the demand flexibility parameters
+    demand_flexibility["enabled"] = true
+    demand_flexibility["duration"] = nothing
+    demand_flexibility["interval_balance"] = true
+    demand_flexibility["rolling_balance"] = true
+
+    # Try loading the demand flexibility parameters
+    demand_flexibility_parameters = DataFrames.DataFrame()
     try
-        demand_flexibility["flex_amt"] = CSV.File(
-            joinpath(filepath, "demand_flexibility.csv")
+        demand_flexibility_parameters = CSV.File(
+            joinpath(filepath, "demand_flexibility_parameters.csv")
         ) |> DataFrames.DataFrame
-        println("...loading demand flexibility profiles")
-        demand_flexibility["enabled"] = true
+        println("...loading demand flexibility parameters")
+
+        # Create a dictionary to hold the error messages relevant to loading the demand 
+        # flexibility parameters
+        demand_flexibility_params_errs = Dict()
+        demand_flexibility_params_errs["duration"] = (
+            "The demand flexibility duration parameter is not defined. Will default to "
+            * "nothing."
+        )
+        demand_flexibility_params_errs["enabled"] = (
+            "The parameter that indicates if demand flexibility is enabled is not "
+            * "defined. Will default to being enabled."
+        )
+        demand_flexibility_params_errs["interval_balance"] = (
+            "The parameter that indicates if the interval load balance constraint is "
+            * "enabled is not defined. Will default to being enabled."
+        )
+        demand_flexibility_params_errs["rolling_balance"] = (
+            "The parameter that indicates if the rolling load balance constraint is "
+            * "enabled is not defined. Will default to being enabled."
+        )
+
```

Comment:
Same comment as above. Also, would be better to call it `warns` instead of `errs` following the convention that an error will stop the program?

Reply:
Yeah, I'll make those changes. Good call on the name change

Reply:
Done.